src/main/firstLaunch.ts

```diff
@@ -37,6 +37,8 @@ export function createFirstLaunchTour() {
 
     makeLinksOpenExternally(win);
 
+    if (PORTABLE && !existsSync(DATA_DIR)) mkdirSync(DATA_DIR);
```

Comment:
i think this call would make more sense directly under where DATA_DIR is defined

also `if (!exists(file)) create(file)` is an antipattern since it could potentially lead to a race condition if the file is deleted/created by a different process between these two checks. it's better to just yolo the action and handle errors. or in the case of mkdir, you can just use the recursive option instead:
```suggestion
    mkdirSync(DATA_DIR, { recursive: true });
```